### PR TITLE
Updated Readme, removed several grammatical errors

### PR DESCRIPTION
**Pull request by urvishajain**
Removed all grammatical errors for example: working: work, moja: Moja, edication: education, punctuation mistakes, article missing, etc.

# Pull Request Template

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,15 +100,15 @@ Moja global welcomes contributions to the community website. If you have an idea
...
 
 ```sh
 yarn lint
 ```
-This command logs all warnings and lint errors and scope of fixes/improvements.
+This command logs all warnings and lint errors and the scope of fixes/improvements.
 
 ```sh
-yarn lint:fix
+yarn lint: fix
```
Comment (sohamsshah) [inline README.md:111]: This is a command `yarn lint:fix`. We can remove the space in between the command.

File: README.md [hunk 2]
```diff
@@ -143,10 +143,10 @@ Moja global thrives on community participation, and we appreciate your contribut
 
 You can reach out to us through the following channels:
 
-- [info@moja.global](mailto:info@moja.global) : Ask questions about moja global relevant to the members and management with a point of focus for new user countries and policy makers.
-- [tsc@moja.global](mailto:tsc@moja.global) : Ask questions about contributing code, helping out with development, resolving issues and building new features with moja global teams.
-- [mentorship@moja.global](mailto:mentorship@moja.global) : Ask questions about participating in the community, sponsoring programs, funding opportunity and community development.
-- [Slack community](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw ): Connect with other developers, scientists, policy makers and other interested people over our Slack community.
+- [info@moja.global](mailto:info@moja.global) : Ask questions about Moja global relevant to the members and management with a point of focus for new user countries and policymakers.
+- [tsc@moja.global](mailto:tsc@moja.global) : Ask questions about contributing code, helping out with development, resolving issues and building new features with Moja global teams.
```
Comment (sohamsshah) [inline README.md:147]: On the moja global organization, across all the places; moja global should be spelled as `moja global` only without Capital Case in the word. So, we shall consider writing `moja global` only everywhere. Please make this change.

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Hi @urvishajain , I have suggested few changes. After that PR is good to go!

File: README.md (hunk 2 above)
Comment (urvishajain) [reply README.md:147]: okay will change it for sure, thanks

Comment (urvishajain) [reply README.md]: Please review the code now.

**Review by sohamsshah — APPROVED** [review]
LGTM!

**Pull request merged**